docs/pages/documentation-overview.mdx

```diff
@@ -0,0 +1,121 @@
+sessions, you can configure RBAC, add resources, and protect your home lab with
+Teleport.
+
+Once you are ready to learn more about Teleport, read our [Core Concepts
+guide](./core-concepts.mdx), which introduces the components of a Teleport
+cluster. You can refer to this glossary as you continue through the
+documentation.
+
+## Critical guides
+
+Guides in the "Home" section include information that all Teleport users will
+probably need to consult at some point:
+
+- [Installation](./installation.mdx): How to install Teleport binaries on your
+  environment. If you are just getting started with Teleport, we recommend
+  spinning up a [demo cluster](./index.mdx) or signing up for a [Teleport Team
+  trial](https://goteleport.com/signup).
+- [Frequently Asked Questions](./faq.mdx): If this page does not answer your
+  question, try our AI-assisted search box on the left sidebar.
+- [Usage Reporting and Billing](./usage-billing.mdx): How Teleport calculates
+  usage metrics to determine customer billing.
+- [Upcoming Releases](./upcoming-releases.mdx): Features to be introduced in
+  future versions of Teleport.
+- [Changelog](./changelog.mdx): A comprehensive collection of release notes.
+- [Teleport Assist](./ai-assist.mdx): How to use the AI-guided Teleport client.
+
+## Choose an edition
+
+After trying out Teleport, you are ready to deploy a cluster to your
+infrastructure. Teleport has four editions: 
+
+- Teleport Team
+- Teleport Enterprise Cloud
+- Teleport Enterprise
+- Teleport Community Edition
+
+You can compare these in our [Choose an
+Edition](./choose-an-edition/introduction.mdx) section.
+
+## Deploy a cluster
+
+Once you know which edition you would like to deploy, read our [Deploy a
+Cluster](./deploy-a-cluster/introduction.mdx) documentation for how to launch a
+fully fledged Teleport cluster in production. (If you are using Teleport
+Enterprise Cloud, you can skip this step.) This section shows you the best
+practices to follow for a high-availability Teleport cluster, and how to deploy
+Teleport on your cloud provider of choice.
+
+## Manage access
+
+Now that you have a running Teleport cluster, set up role-based access controls
+to enable secure access to your infrastructure.  You can define roles with
+granular permissions and use Teleport's integrations with Single Sign-On
+providers to automatically map these roles to users. You can also set up Access
+Requests to enable just-in-time access to your infrastructure. Read [Manage
+Access](./access-controls/introduction.mdx) to get started.
+
+## Manage your cluster
+
+With your Teleport cluster configured, you can now begin Day Two operations
+such as upgrades, adding agents to the cluster, and integrating Teleport with
+third-party tools. Read [Manage your
+Cluster](./management/introduction.mdx) for more information.
+
+## Add your infrastructure
+
+Teleport is protocol aware and provides functionality that is unique to each
```

Comment:
```suggestion
Teleport is protocol-aware and provides functionality that is unique to each
```

Reply:
It's pretty common in style guides to allow for excluding hyphens in compound modifiers that take place after nouns, e.g., https://www.grammarly.com/blog/hyphen/